sphere-common/src/main/java/io/sphere/sdk/search/TermFacetExpression.java

```diff
@@ -5,6 +5,7 @@
 import static java.util.Collections.emptyList;
 
 public class TermFacetExpression<T, V> extends TermExpression<T, V> implements FacetExpressionBase<T> {
+    SearchModel<T> model;
```

Comment:
you give the search model to the super class constructor, why do you add a field in the subclass which looks unused?